src/Uno.Extensions.Authentication.MSAL/MsalAuthenticationProvider.cs

```diff
@@ -140,6 +140,15 @@ private async Task SetupStorage()
 			if (Logger.IsEnabled(LogLevel.Trace)) Logger.LogTraceMessage($"Setting up storage location");
 
 			var folderPath = await Storage.CreateFolderAsync(Name.ToLower());
+			if (folderPath is null)
+			{
+				if (Logger.IsEnabled(LogLevel.Warning))
+				{
+					Logger.LogWarningMessage($"Folder should not be null, exiting Msal storage setup");
```

Comment:
What can the user do when this error happens? Can the message be more actionable?

Reply:
In theory this should never happen. Currently there are some scenarios, such as with Linux Framebuffer, where appdata folder is null but this is being addressed in the breaking changes work